spec/cohort.spec.js

```diff
@@ -0,0 +1,227 @@
+import { Cohort } from '../src/cohort.js'
+
+describe('createCohort', () => {
+  let cohort
+  beforeEach(() => {
+    cohort = new Cohort()
+  })
+
+  it('creates a new cohort object with a cohortName property', () => {
+    const cohortName = 'cohort_01'
+    const cohortObject = {
+      cohortName: cohortName,
+      studentList: []
+    }
+    const cohortList = [cohortObject]
+    const res = cohort.createCohort('cohort_01')
+    expect(res).toEqual(cohortList)
+  })
+
```

Comment:
My favourite part of this is having blank lines between it statements <3 it becomes very readable

Reply:
Yeah I was going mad with them all pushed together!